modin/pandas/base.py

```diff
@@ -3165,17 +3205,21 @@ def __get_groups(
     @property
     def groups(self):
         return self._query_compiler.default_to_pandas(
-            lambda df: pandas.DataFrame.resample(df, *self.resample_args).groups
+            lambda df: pandas.DataFrame.resample(
+                df, *self.resample_args.values()
```

Comment:
Should we pass an exact list of arguments to the QC and further because there is an incompatibility between dict.values() and list?

Reply:
Generally, it would be better to pass dict of Resampler kwargs to the QC and further. Issue related this is #3724.

I agree that using of `values` instead of list before could make effects due to changed order of arguments in dict.

Reply:
I suggest moving to the use of dict in #3724 and not introducing the logic here. Just let's keep a list as it was.

Reply:
Now self.resample_args is list